docs/frameworks/mastra.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: "Mastra"
+description: "Use Humanlayer with Mastra, the TypeScript AI Framework"
+icon: "solar-system"
+---
+
+## Overview
+
+[Mastra](https://mastra.ai) is a TypeScript framework for building AI applications. HumanLayer adds human oversight to your AI features.
+
+## Getting Started with Mastra
+
+### Installation
+
+The easiest way to get started with Mastra is to use `create-mastra`:
+
+```bash
+npx create-mastra@latest
+```
+
+This command will guide you through setting up a new Mastra project with all the necessary configurations.
+
+<Tip>
+  For more installation and project set up details, check Mastra's documentation
+  [here](https://mastra.ai/docs)
+</Tip>
+
+### Model Providers
+
+<Info>
+  While Google's Gemini models are used in the examples below, Mastra supports
+  all models available through the AI SDK:
+</Info>
+
+- OpenAI: `import { openai } from "@ai-sdk/openai"`
+- Anthropic: `import { anthropic } from "@ai-sdk/anthropic"`
+- Google: `import { google } from "@ai-sdk/google"`
+- And more...
+
+### Running Your Project
+
+Once your project is set up, you can run it using:
+
+```bash
+npm run dev
+```
+
+This will start your Mastra application, and you can access the playground at:
+
+```
+http://localhost:4111
+```
+
```

Comment:
the default agent seemed to be broken when I ran this workflow - https://www.loom.com/share/d21acab6278f4ac9b788998a2fd1bd5f